docs/source/routing/get-started.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Apollo Router Quickstart
+subtitle: Run a router binary
+description: This quickstart tutorial walks you through installing an Apollo Router binary, running it with an example supergraph schema and a YAML configuration file, and making test queries with Apollo Sandbox.
+---
+
+import ElasticNotice from "../../shared/elastic-notice.mdx";
+
+Hello! Let's run Apollo Router for the first time, using the simple scenario of developing and deploying locally.
+
+In this guide, you will:
+
+- Download and install a router binary.
+- Download a supergraph schema.
+- Create a router YAML configuration file.
+- Run the router in development mode.
+- Make a query to the running router.
+
+## 1. Install router binary
+
+Let's start by downloading and installing a router binary locally.
+
+1. Download and install the latest version of the router binary with a single command line:
+
+    ```bash showLineNumbers=false
+    curl -sSL https://router.apollo.dev/download/nix/latest | sh
+    ```
+
+    <ExpansionPanel title="Download a specific router version">
+
+    To download and install a specific version of router, set the version in the download URL path.
+
+    For example, to download router v2.0.0:
+
+    ```bash showLineNumbers=false
+    curl -sSL https://router.apollo.dev/download/nix/v2.0.0 | sh
+    ```
+
+    </ExpansionPanel>
+
+    Optionally, go to [router releases](https://github.com/apollographql/router/releases) in GitHub to download and extract a bundle.
+
+1. Check that your router installed successfully by running the `router` binary from your project's root directory:
+
+    ```bash showLineNumbers=false
+    ./router --version
+    ```
+
+
+## 2. Download supergraph schema
+
+A router needs a schema for the federated graph, or "supergraph", that it's orchestrating. For this guide, we're downloading an example supergraph schema, which we'll provide to the router when running it.
```

Comment:
Conventionally we've used italics for key terms. Additionally, beyond the opening "Let's", you've used the second person. Try:

```suggestion
A router needs a schema for the federated graph, or _supergraph_, that it's orchestrating. This guide includes an example supergraph schema you can download and provide to the router.
```

Reply:
```suggestion
A router needs a schema for the federated graph, or _supergraph_, that it's orchestrating. This guide uses an example supergraph schema, which you download and provide to the router.
```